impeller/renderer/backend/gles/blit_command_gles.cc

```diff
@@ -372,6 +372,8 @@ bool BlitResizeTextureCommandGLES::Encode(const ReactorGLES& reactor) const {
     return false;
   }
 
+  destination->SetCoordinateSystem(TextureCoordinateSystem::kUploadFromHost);
```

Comment:
Rather than setting the destination to uplaod from host, we should set it to whatever the source is.

Reply:
done